src/query/execute_schema.c

```diff
@@ -10016,6 +10020,366 @@ do_alter_change_tbl_comment (PARSER_CONTEXT * const parser, PT_NODE * const alte
   return error;
 }
 
+/*
+ * do_alter_change_col_comment() - change the column comment
+ *   return: Error code
+ *   parser(in): Parser context
+ *   alter(in/out): Parse tree of a PT_CHANGE_COLUMN_COMMENT clause
+ */
+static int
+do_alter_change_col_comment (PARSER_CONTEXT * const parser, PT_NODE * const alter)
+{
+  int error = NO_ERROR;
+  const PT_ALTER_CODE alter_code = alter->info.alter.code;
+  const char *entity_name = NULL;
+  DB_OBJECT *class_obj = NULL;
+  DB_CTMPL *ctemplate = NULL;
+  SM_ATTR_CHG_SOL change_mode = SM_ATTR_CHG_ONLY_SCHEMA;
+  SM_ATTR_PROP_CHG attr_chg_prop;
+  bool tran_saved = false;
+  MOP class_mop = NULL;
+  OID *usr_oid_array = NULL; //--
+  int user_count = 0;
+  int i;
+  bool has_partitions = false; //--
+  bool is_srv_update_needed = false;
+  OID class_oid;
+  int att_id = -1;
+
+  PT_NODE *attr_node = NULL;
+  PT_NODE *comment_node = NULL;
+  PARSER_VARCHAR *comment = NULL;
+
+  assert (alter_code == PT_CHANGE_COLUMN_COMMENT);
+  assert (alter->info.alter.super.resolution_list == NULL);
+
+  attr_node = alter->info.alter.alter_clause.attr_mthd.attr_def_list;
+  comment_node = attr_node->info.attr_def.comment;
+
+  assert (comment_node != NULL && comment_node->node_type == PT_VALUE);
+
+  comment = comment_node->info.value.data_value.str;
+
+  OID_SET_NULL (&class_oid);
+  reset_att_property_structure (&attr_chg_prop);
+
+  entity_name = alter->info.alter.entity_name->info.name.original;
+  if (entity_name == NULL)
+    {
+      error = ER_UNEXPECTED;
+      er_set (ER_ERROR_SEVERITY, ARG_FILE_LINE, error, 1, "Expecting a class or virtual class name.");
+      goto exit;
+    }
+
+  class_obj = db_find_class (entity_name);
+  if (class_obj == NULL)
+    {
+      assert (er_errid () != NO_ERROR);
+      error = er_errid ();
+      goto exit;
+    }
+
+  error = locator_flush_class (class_obj);
+  if (error != NO_ERROR)
+    {
+      /* don't overwrite error */
+      goto exit;
+    }
+
+  /* force exclusive lock on class, even though it should have been already acquired */
+  if (locator_fetch_class (class_obj, DB_FETCH_WRITE) == NULL)
+    {
+      error = ER_FAILED;
+      goto exit;
+    }
+
+  ctemplate = dbt_edit_class (class_obj);
+  if (ctemplate == NULL)
+    {
+      /* when dbt_edit_class fails (e.g. because the server unilaterally aborts us), we must record the associated
+       * error message into the parser.	 Otherwise, we may get a confusing error msg of the form: "so_and_so is not a
+       * class". */
+      pt_record_error (parser, parser->statement_number - 1, alter->line_number, alter->column_number, er_msg (), NULL);
+      error = er_errid ();
+      goto exit;
+    }
+
+  error =
+    check_change_attribute (parser, ctemplate, alter->info.alter.alter_clause.attr_mthd.attr_def_list,
```

Comment:
not needed the function **check_change_attribute** to be modified to newly add comment attribute?